builtins/safe-default-configuration.txt

```diff
@@ -243,3 +243,211 @@ math semantics
 - mathsize
 - scriptlevel
 
+
+// SVG 2
+// https://svgwg.org/svg2-draft/
+// as referenced here: https://html.spec.whatwg.org/#refsSVG
+
+// Document structure, https://svgwg.org/svg2-draft/struct.html
+svg svg
+- viewBox
+- preserveAspectRatio
+- transform
+- height  // Specified as global "presentation attribute"
+- width  // Specified as global "presentation attribute"
+- x  // Specified as global "presentation attribute"
+- y  // Specified as global "presentation attribute"
+svg g
+svg defs
+svg symbol
+- preserveAspectRatio
+- viewBox
+- refX
+- refY
+- height  // Specified as global "presentation attribute"
+- width  // Specified as global "presentation attribute"
+- x  // Specified as global "presentation attribute"
+- y  // Specified as global "presentation attribute"
+
+// Purposely omitted: svg use.
+svg switch
+svg title
+svg desc
+svg metadata
+
+// Paths, https://svgwg.org/svg2-draft/paths.html
+svg path
+- pathLength
+- d
+
+// Shapes, https://svgwg.org/svg2-draft/shapes.html
+svg rect
+- pathLength
+- x
+- y
+- width
+- height
+- rx
+- ry
+svg circle
+- pathLength
+- cx
+- cy
+- r
+svg ellipse
+- pathLength
+- cx
+- cy
+- rx
+- ry
+svg line
+- pathLength
+- x1
+- y1
+- x2
+- y2
+svg polyline
+- pathLength
+- points
+svg polygon
+- pathLength
+- points
+// Text, https://svgwg.org/svg2-draft/text.html
+svg text
+- lengthAdjust
+- x
+- y
+- dx
+- dy
+- rotate
+- textLength
+svg tspan
+- lengthAdjust
+- x
+- y
+- dx
+- dy
+- rotate
+- textLength
+svg textPath
+- lengthAdjust
+- textLength
+- path
+// - href, purposely omitted: This loads a textPath from a resource.
+- startOffset
+- method
+- spacing
+- side
+
+// Embedded Content, https://svgwg.org/svg2-draft/embedded.html
+svg image
```

Comment:
Not sure if `<svg:image>` without `href` attribute still makes sense...

Reply:
I don't think so. We also don't have `<img>`, so I don't think we should have this either.

Reply:
Removed.